engine/engine/content/builtins/render/default.render_script

```diff
@@ -97,45 +182,42 @@ function update(self)
     render.set_blend_func(render.BLEND_SRC_ALPHA, render.BLEND_ONE_MINUS_SRC_ALPHA)
     render.disable_state(render.STATE_CULL_FACE)
 
-    render.draw(self.tile_pred, {frustum = frustum})
-    render.draw(self.particle_pred, {frustum = frustum})
+    render.draw(predicates.tile, camera_world.frustum)
+    render.draw(predicates.particle, camera_world.frustum)
```

Comment:
Should we also draw models?

Reply:
Not sure, what do you think? Maybe we can do it as a separate step lately when we decide how we wanna work with camera and render_script modularity

Reply:
> Not sure, what do you think? Maybe we can do it as a separate step lately when we decide how we wanna work with camera and render_script modularity

I think you should draw models as well! So that the standard material for models will work on the standard render script. Plus, it will be possible to simplify 3d template by removing its version of render script from it. This will make the engine much more 3D friendly.

Since the PR was accepted, should I write a feature request?

Reply:
>Since the PR was accepted, should I write a feature request?

yes, pls
It's easier to do one thing at a time, and solve one problem at a time. That's why I didn't want to add it as part of this PR

Reply:
https://github.com/defold/defold/issues/8158